infra/auth0/js/setCustomIdTokenClaims.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Handler that will be called during the execution of a PostLogin flow.
+ *
+ * @param {Event} event - Details about the user and the context in which they are logging in.
+ * @param {PostLoginAPI} api - Interface whose methods can be used to change the behavior of the login.
+ */
+exports.onExecutePostLogin = async (event, api) => {
+  const namespace = "https://online.ntnu.no"
+
+  const { study_year, ow_user_id } = event.user.app_metadata
+  const { middle_name } = event.user.user_metadata
+
+  if (study_year) {
+    api.idToken.setCustomClaim(`${namespace}/study_year`, study_year)
+  }
+
+  if (ow_user_id) {
+    api.idToken.setCustomClaim(`${namespace}/ow_user_id`, ow_user_id)
```

Comment:
Hvilke OW er dette ID-en til?

Reply:
Ow5

Reply:
Hvorfor har OW5 egne bruker-ID-er? bestemte vi oss ikke for at vi bare bruke Auth0-sub-en som ID?

Reply:
For å decouple brukerene våre fra auth0. Vi snakket om det ja, men jeg og Mats ble enige om at det er nice å ikke fylle databasen med user FK's som er auth0-id'er